pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -255,11 +261,26 @@ func (r *Reconciler) reconcileDeployment(logger *zap.SugaredLogger, el *v1alpha1
 		return err
 	}
 
-	labels := mergeMaps(el.Labels, GenerateResourceLabels(el.Name))
 	var replicas = ptr.Int32(1)
 	if el.Spec.Replicas != nil {
 		replicas = el.Spec.Replicas
 	}
+	var env []corev1.EnvVar
+	if el.Spec.Resources.KubernetesResource != nil {
+		if len(el.Spec.Resources.KubernetesResource.Template.Spec.Containers) == 1 {
+			for i := range el.Spec.Resources.KubernetesResource.Template.Spec.Containers[0].Env {
```

Comment:
Being able to specify extra env vars for the EL sink is a new feature. Was there a specific use case you had in mind?

Reply:
Oh ya this will be considered as new feature based on usecase because currently we don't allow.

 > Was there a specific use case you had in mind?

No specific usecase i just added while coding.

But yes i will remove it now and we can add it later if we get any usecase for this :+1:

Reply:
Removed env support.